src/components/DataTable/components/Cell/Cell.tsx

```diff
@@ -71,7 +72,7 @@ function Cell({
   };
 
   const direction = sorted ? sortDirection : defaultSortDirection;
-  const source = `caret${direction === 'ascending' ? 'Up' : 'Down'}`;
```

Comment:
String interpolation like this should be illegal

Reply:
💯